BraveShared/BraveStrings.swift

```diff
@@ -2096,7 +2096,7 @@ extension Strings {
         public static let shortcutSettingsOpenPlaylistDescription =
             NSLocalizedString("shortcuts.shortcutSettingsOpenPlaylistDescription",
                               bundle: .braveShared,
-                              value: "Use Shortcuts to Open Playlist via Siri - Voice Assistant",
+                              value: "Use Shortcuts to open playlist via Siri - Voice Assistant",
```

Comment:
Let's capitalize `Playlist` here. It's our product and we sometimes capitalize it. It is not consistent across the app unfortunately.

Reply:
Done